x/launch/client/cli/tx_settle_request.go

```diff
@@ -0,0 +1,50 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+func CmdSettleRequest() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "settle-request [approve/reject] [chainID] [requestID]",
+		Short: "Approve or reject a pending request",
+		Args:  cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			approve, err := strconv.ParseBool(args[0])
```

Comment:
I meant in my previous comment that I think it can be more explicit to either provide `"approve"` or `"reject"` string as the first argument instead of using `ParseBool`  and returning an error if none is provided